app/assets/yge/youvegoteth/send.js

```diff
@@ -43,40 +44,54 @@ var updateEstimate = function(e) {
 
 };
 
-window.onload = function() {
-  jQuery('#amount').on('keyup blur change', updateEstimate);
-  jQuery('#token').on('change', updateEstimate);
-
-  unPackAddresses();
-
-  var min_send_amt_wei = 6000000;
+function isSendTipPage() {
+  return (/\/tip\/send/).test(window.location.pathname);
+}
 
+function prePopulateFunderFields() {
   if (localStorage['amount']) {
     $('amount').value = localStorage['amount'];
   }
-  if (localStorage['username']) {
-    $('username').value = localStorage['username'];
-  }
   if (localStorage['issueURL']) {
     $('issueURL').value = localStorage['issueURL'];
   }
-  if (localStorage['email']) {
-    $('email').value = localStorage['email'];
-  }
   if (localStorage['fromName']) {
     $('fromName').value = localStorage['fromName'];
   }
   if (localStorage['fromEmail']) {
     $('fromEmail').value = localStorage['fromEmail'];
   }
+  if (localStorage['expires']) {
+    $('expires').selectedIndex = localStorage['expires'];
+  }
+}
+
+function prePopulateGitcoinerFields() {
+  if (localStorage['username']) {
+    $('username').value = localStorage['username'];
+  }
+  if (localStorage['email']) {
+    $('email').value = localStorage['email'];
+  }
   if (localStorage['comments_priv']) {
     $('comments_priv').value = localStorage['comments_priv'];
   }
   if (localStorage['comments_public']) {
     $('comments_public').value = localStorage['comments_public'];
   }
-  if (localStorage['expires']) {
-    $('expires').selectedIndex = localStorage['expires'];
+}
+
+window.onload = function() {
+  jQuery('#amount').on('keyup blur change', updateEstimate);
+  jQuery('#token').on('change', updateEstimate);
+
+  unPackAddresses();
+
+  var min_send_amt_wei = 6000000;
+
+  prePopulateFunderFields();
+  if (!isSendTipPage) {
```

Comment:
@arun1595 `isSentTipPage()` ?

Reply:
wait didn't you add `isSendTipPage` function [here](https://github.com/gitcoinco/web/pull/1318/files#diff-69887365dcfb83895cb00a1c9639f716R47) ? 🤔 
It looks like you wanted it to populate the gitcoinerField when the url isn't `/funding/new`

Also `isSentTipPage` is a function so wouldn't you need to invoke it rather than simply using it ?

Reply:
We don't want to populate gitcoiner fields if the current page is `/tip/send` right?

Derp, didn't understand what you meant in your other comment, fixed.